.circleci/config.yml

```diff
@@ -270,17 +288,16 @@ jobs:
             - 'ed:c9:40:2a:96:6c:90:fd:46:18:2d:bf:8d:a8:a9:b3'
       - run: ./deploy.sh << parameters.public_path >>
 
-  publish_artifact:
+  publish_on_npm_registry:
     executor: base-executor
-    working_directory: << pipeline.parameters.working_directory >>/raiden-ts
+    working_directory: << pipeline.parameters.working_directory >>
     steps:
       - attach_workspace: *attach_options
-      - run:
-          name: Authenticate with registry
-          command: echo "//registry.npmjs.org/:_authToken=$NPM_AUTH_TOKEN" > ~/.npmrc
-      - run:
-          name: Publish on npm
-          command: yarn publish --no-git-tag-version --non-interactive
+      - publish_workspace_on_npm_registry:
+          workspace: 'raiden-ts'
+      - publish_workspace_on_npm_registry:
+          workspace: '@raiden_network/raiden-cli'
+          tarball: 'raiden-cli.tgz'
```

Comment:
Hmm, you're publishing the bundle here? This artifact is expected to be used standalone, with just `node` as dependency. It doesn't require the full dependencies of the package to run. Since `npm -g` or `yarn global` will install the dependencies anyway, I think you shoud/could be publishing `build` instead, which should be lighter and allow e.g. dependencies switching depending on who is installing it, etc, which wouldn't work ootb with the bundle.

Reply:
Hmm. Okay. I need to give this a practical check. If it works I'm fine with it.

Reply:
Yarn and the NPM registry have issue atm...

Reply:
Okay now it worked (cleared Yarn its cache). I will push a commit with the changes.

Reply:
✔️